app/views/find_local_council/one_council.html.erb

```diff
@@ -18,10 +25,19 @@
           href: @authority["homepage_url"],
           text: t("formats.local_transaction.local_authority_website", local_authority_name: @authority["name"]),
           data_attributes: {
-            module: "gem-track-click",
+            module: "gem-track-click ga4-link-tracker",
             track_category: "pageElementInteraction",
             track_action: "unitaryLinkClicked",
-            track_label: @authority["homepage_url"]
+            track_label: @authority["homepage_url"],
+            ga4_link: {
+              event_name: "information_click",
```

Comment:
The implementation guide also has `'action':'information click',` - should this be added?

Reply:
Added.